packages/popover2/src/popover2.tsx

```diff
@@ -685,7 +685,7 @@ export class Popover2<T extends DefaultPopover2TargetHTMLProps> extends Abstract
             return;
         }
 
-        const eventTarget = e.target as HTMLElement;
+        const eventTarget = (e.composed ? e.composedPath()[0] : e.target) as HTMLElement;
```

Comment:
It makes sense according to e is "React.SyntheticEvent". But actually the caller of this function passes native event instead of React.SyntheticEvent.
I guess in that case I can check that "e" has "nativeEvent" and use it, otherwise use "e"

Reply:
So I updated the code